reports/post_diagnostics.Rmd

```diff
@@ -60,6 +99,7 @@ kable(log_status %>%
     column_spec(1, border_right = "1px solid #d2d2d2") %>% 
     row_spec(which(log_status$status == "ERROR"), background = "#ffa07a") %>% 
     scroll_box(height = "300px")
+
```

Comment:
Would you mind removing the extra space? We haven't been doing this in other sections in the same file.